stdlib/3/builtins.pyi

```diff
@@ -582,6 +582,7 @@ class slice:
     def indices(self, len: int) -> Tuple[int, int, int]: ...
 
 class tuple(Sequence[_T_co], Generic[_T_co]):
+    def __new__(cls: Type[_T], iterable: Iterable[Any]=tuple()) -> _T: ...
```

Comment:
The default should always be `...`. You should basically match `__init__` here: take `Iterable[_T_co]` and return `_T[_T_co]`.